library/ssl_msg.c

```diff
@@ -643,28 +643,19 @@ int mbedtls_ssl_encrypt_buf( mbedtls_ssl_context *ssl,
             return( MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL );
         }
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
-        if( transform->minor_ver >= MBEDTLS_SSL_MINOR_VERSION_3 )
-        {
-            unsigned char mac[MBEDTLS_SSL_MAC_ADD];
+        unsigned char mac[MBEDTLS_SSL_MAC_ADD];
```

Comment:
Note on commit structure: the commit message says "Reverting deleted macros " but the commit also contains other changes (remove useless version checks). It would have been better to have two commits here.

No change required here, just mentioning so that you can keep it in mind for future PRs. In general, commits should do just one thing. For example the first commit in the PR is "Remove all TLS 1.0 and 1.1 instances and add some compatibility tests" and should have been two commits too (1. remove, 2. add tests), as the word "and" in the description suggests :)

It's good to group small/trivial things in a single commit if they have a common theme, for example "Fix typos" or "Improve comments". But "address review feedback" doesn't work as a common theme: the commit message should make sense even without looking at the discussion on github.